packages/client/hmi-client/src/workflow/ops/model/tera-model-node.vue

```diff
@@ -129,4 +129,8 @@ main {
 	border-radius: var(--border-radius);
 	overflow: hidden;
 }
+
+:deep(.smallSegmentedControl .p-button-label) {
+	font-size: var(--font-caption);
+}
```

Comment:
classes should be kebab case. If we want to globalize this rule you can add this to `extensions/_selectbutton.scss`

```
.p-selectbutton.p-button-xsm {
    @extend .p-button-sm;
    .p-button-label {
        font-size: var(--font-caption);
    }
}
```

and then`.p-button-xsm` would be the only class needed in the `SelectButton`

Reply:
I don't really understand... if there's a better way to do this feel free to adjust it

Reply:
Sure, I'll take care of it